content/docs/user-guide/basic-concepts/pipeline.md

```diff
@@ -1,7 +1,10 @@
 ---
-name: Pipeline (DAG)
-match: [DAG, pipeline, 'data pipeline', 'data pipelines']
+name: Pipeline
+match: [pipeline, pipelines, 'data pipeline', 'data pipelines']
 tooltip: >-
-  A set of inter-dependent stages. This is also called a [dependency
-  graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+  A set of inter-dependent stages defined in
```

Comment:
Since we are touching here - I think we should be more high level here (what problem do they solve - structure your project in a declarative way for reproducibility, for agility (caching results), standardize workflow, etc ...

Reply:
The glossary tooltips are meant as a quick definition, there's a link to this page from there for more context. I wouldn't address that change in this PR if we want to change the purpose of the tooltips. It was changed because I removed most links to Wikipedia and instead linked to the DAG section in this new guide.

Reply:
yep, that's fine. Also, no need to change the purpose. It should be a quick definition. It's just the current one is not very meaningful and is hard to understand if you are not familiar with DVC pipelines.

Reply:
OK how about this? Committed for now

<img width="633" alt="image" src="https://user-images.githubusercontent.com/1477535/186266417-ff0cc018-7bee-4605-ad62-182dcd081b0e.png">

Reply:
yep, way better!